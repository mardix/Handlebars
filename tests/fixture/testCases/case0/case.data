Organization.Address=123 Main Street
Recipient.City=Atlanta
Recipient.Company=Salesforce.com Inc.
Recipient.Country=USA