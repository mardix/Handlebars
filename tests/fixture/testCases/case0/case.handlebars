{{Organization.Address}}
{{Recipient.City}}
{{Recipient.Company}}
{{Recipient.Country}}