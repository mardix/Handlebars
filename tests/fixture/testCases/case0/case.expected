123 Main Street
Atlanta
Salesforce.com Inc.
USA